timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/database/TimbuctooActionsStubs.java

```diff
@@ -7,7 +7,11 @@
 import java.time.Clock;
 
 import static org.mockito.Mockito.mock;
-  
+
+/**
+ * @deprecated mock TimbuctooActions or for a database integration test create one with the constructor.
+ */
+@Deprecated
```

Comment:
typo in the commit message

also: I'm not sure that I want people to use the constructor in their tests.